chia/util/db_wrapper.py

```diff
@@ -145,6 +145,7 @@ async def create(
         synchronous: Optional[str] = None,
         foreign_keys: bool = False,
         row_factory: Optional[Type[aiosqlite.Row]] = None,
+        soft_heap_limit: Optional[int] = None,
```

Comment:
If this is going to be optional then perhaps we shouldn't default to 0 when pulling out of the config?  And `null` should be encouraged, and used, in the config?

Reply:
RIght now, I'm thinking 0 might be better. although I'm a little confused in that sqlite seems to remember this pragma across test runs in some way. And 0 resets the limit back to no-limit, so 0 I think might be better.